### PR TITLE
Fix FMW integration tests for OKD 

### PR DESCRIPTION
**Pull request by maggiehe00**
OKD Jenkins job passed at https://build.weblogick8s.org:8443/job/wko-okd-test/22/testReport/

Kind New Jenkins job result after merging to the latest main:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/6541/testReport/
1 failure: https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/6569/testReport/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInPV.java [hunk 1]
```diff
@@ -148,7 +148,11 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
   void testFmwDynamicDomainInPV() {
     // create FMW dynamic domain and verify
     createFmwDomainAndVerify();
-    verifyDomainReady();
+    //verifyDomainReady();
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInPV.java:151]: Remove commented code

Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInPV.java]: Done

File: integration-tests/src/test/resources/bash-scripts/setup_wdt.sh [hunk 2]
```diff
@@ -79,6 +79,9 @@ SCRIPTPATH="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
 WDT_MODEL_FILE=${WDT_MODEL_FILE:-"$SCRIPTPATH/wdt_model.yaml"}
 WDT_VAR_FILE=${WDT_VAR_FILE:-"$SCRIPTPATH/create-domain-inputs.yaml"}
 
+CDIR=`pwd`
+echo "CDIR = $CDIR"
+echo "SCRIPTPATH = $SCRIPTPATH"
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/resources/bash-scripts/setup_wdt.sh:84]: Can you remove the above 3 lines? I added that when trying to debug

**Review by bhavaniravichandran — APPROVED** [review]
minor comments. Good to go.

File: integration-tests/src/test/resources/bash-scripts/setup_wdt.sh (hunk 2 above)
Comment (maggiehe00) [reply integration-tests/src/test/resources/bash-scripts/setup_wdt.sh]: Done

**Review by anpanigr — APPROVED** [review]
LGTM

**Pull request merged**